EclipseCon2023/ClusterStacks.md

```diff
@@ -0,0 +1,64 @@
+# EclipseCon 2023
+
+## Session Title
+Create sovereign container platforms with SCS ClusterStacks
+
+## Speakers
+- Janis Kemper
+- Jan Schoone
+
+## Audience
+### Session Topic
+Open Source Best Practices
+
+### Session Type
+- [x] Standard (30 minutes)
+- [ ] Tutorial (3 hours)
+
+### Experience level
+- [ ] Beginner
+- [x] Intermediate
+- [ ] Advanced
+
+### Attendee pre-requisites - If none, enter "N/A"
+N/A
+
+### Speaking Experiences
+N/A
+
+### Speaking References
+N/A
+
+## Objectives of the presentation
+Attendees will be enabled to create, update and delete SCS compliant
+standardized Kubernetes clusters on various cloud providers using ClusterAPI
+based ClusterStacks
```

Comment:
Furthermore awareness for the necessity for digital sovereign infrastructure is raised.